src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -151,6 +151,15 @@ class oopDesc;
 #define INTX_FORMAT_W(width)  "%" #width PRIdPTR
 #define UINTX_FORMAT_W(width) "%" #width PRIuPTR
 
+// Convert pointer to intptr_t, for use in printing pointers.
+inline intptr_t p2i(const void* p) {
```

Comment:
I don't think the `const void*` overload is needed, given the presence of the `const volatile void*` overload.